docs/gallery/interactive/pyfar_plotting.ipynb

```diff
@@ -0,0 +1,409 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Plots and styling using pyfar\n",
+    "\n",
+    "Plotting and visualizing data is an integral part of data analysis.\n",
+    "pyfar provides functions to create time signals, frequency spectra, and other related data plots.\n",
+    "These can be used to inspect the data or generate scientific plots.\n",
+    "\n",
+    "Under the hood, pyfar uses the matplotlib library for plotting.\n",
+    "This allows for a high degree of customization in the creation of figures.\n",
+    "This notebook provides an overview of the plotting functions and how they can be used to customize the plots for your use-cases.\n",
+    "For more details on the plotting functions, please refer to the [documentation](https://pyfar.readthedocs.io/en/stable/modules/pyfar.plot.html).\n",
+    "\n",
+    "pyfar also adds interactive features to the plots for inspection.\n",
+    "For example, zooming and cycling through channels, to name a few.\n",
+    "This can be tested in [this](./pyfar_interactive_plots.ipynb) notebook."
```

Comment:
```suggestion
    "pyfar also adds interactive features to the plots for inspection,",
    "for example, zooming and cycling through channels.\n",
    "This can be tested in [this](./pyfar_interactive_plots.ipynb) notebook."
```